### DISCUSSION TITLE
list environments

### DISCUSSION THREAD

**Discussion by rolfmblindgren** [sile-typesetter/sile#1390] [Ideas]
Has anybody tried to implement the standard LaTeX itemize/enumerate/description environments?

**Comment by alerque** [comment]
Yes. See #1365 and #614. The former of those is likely to end up in the next release.

**Reply by Omikhleia** [reply]
Well, not the "description" environment (similar to "definition lists" in (extended-)Markdown or HTML), but besides the fact that it is  somewhat different in structural nature (they have two parts normally, term and definition), one could also argue it could be considered as a different kind of beast (e.g. these lists seldom need to be nested, at least in my experience).

**Comment by alerque** [comment]
A `lists` package with enumerate and itemize functions landed in v0.13.2 thanks to @Omikhleia.
